packaging/prerelease/pull_build.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -e -u -o pipefail # Fail on error
+
+dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+cd "$dir"
+
+# If you ever update this script you should manually git pull on the
```

Comment:
Huh, I don't get it. Why wouldn't the change to this script be picked up by the git pull and used the next time around?